src/pages/main-page/assets/landing-section.module.scss

```diff
@@ -42,4 +55,111 @@
         height: 100%;
     }
 
+    .sectionWrapper {
+        grid-column: 2 / 3;
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+        align-items: center;
+        margin-top: 230px;
+
+        @include breakpoint-md {
+            margin-top: 200px;
+        }
+
+        .sectionTitle {
+            font-family: $kronaOne;
+            font-size: 2.625rem;
+            letter-spacing: 0.9rem;
+            color: white;
+            text-align: center;
+        }
+
+        .sectionDescription {
+            font-family: $alegreyaSans;
+            font-size: 1.5rem;
+            letter-spacing: 0.5rem;
+            color: white;
+            margin: 25px 0 40px 0;
+            text-align: center;
+        }
+
+        .sectionButtons {
+            margin: 50px 0 110px 0;
+            display: flex;
+            justify-content: space-around;
+            align-items: center;
+            width: 80%;
+
+            .sectionButton {
+                font-family: $alata;
+                font-size: 1.2rem;
+                letter-spacing: 0.07rem;
+                padding: 12px 50px 12px 50px;
```

Comment:
you can replace it with 'padding: 12px 50px;' look for other places we you can do it